docs/swift_code_design.md

```diff
@@ -9,10 +9,18 @@ Bindings generation for Swift consists of the Swift code and some translation la
 
 ## Interfaces with C++ implementation
 
-By default Franca interfaces are generated as classes in Swift and abstract classes in BaseAPI. Calling into C++ objects needs to pass through the C interface. The lifetime of the C++ object can be prolonged by holding a reference on Swift side.
+By default interfaces are generated as classes in Swift and abstract classes in BaseAPI. Calling into C++ objects needs to pass through the C interface. The lifetime of the C++ object can be prolonged by holding a reference on Swift side.
 ![Interfaces code overview](diagrams/SwiftInterfaces.svg)
 
 ## Interfaces with Swift implementation
 
 If the deployment property `IsInterface=true` is set, Gluecodium generates a protocal in Swift which can be implemented in Swift and passed to C++. Gluecodium generates the necessary proxy class. Holding a `shared_ptr` to the proxy on C++ side will extend the life of the Swift object.
```

Comment:
Wow, "deployment property" that's quite vintage. Could you please update this section as well?